ntc_templates/templates/mikrotik_routeros_ip_neighbor_print_detail.textfsm

```diff
@@ -0,0 +1,23 @@
+Value NUM (\d+)
+Value INTERFACE (\S+)
+Value ADDRESS (\S+)
+Value ADDRESS4 (\S+)
```

Comment:
What is the point of this versus ADDRESS?

Reply:
I couldn't find description of this value on the official [documentation](https://help.mikrotik.com/docs/display/ROS/Neighbor+discovery), but it has `address6` value description. So I assume that MikroTik has separate value for IPv4 address also.
Renamed this value to IPV4_ADDRESS